package.json

```diff
@@ -62,6 +62,7 @@
     "lint": "eslint .",
     "precommit": "lint-staged && flow && npm test && npm run prepare",
     "prepare": "husky install && npm-run-all clean -p build.*",
+    "prepare.dev": "husky install && EDGE_MODE=development npm-run-all clean -p build.*",
```

Comment:
Please remove the husky bit, since husky would have already been installed by now.